lib/internal/timers.js

```diff
@@ -545,10 +567,14 @@ function getTimerCallbacks(runNextTicks) {
 
       try {
         const args = timer._timerArgs;
+        let result;
         if (args === undefined)
-          timer._onTimeout();
+          result = timer._onTimeout();
         else
-          timer._onTimeout(...args);
+          result = timer._onTimeout(...args);
+
+        if (result !== undefined && result !== null)
```

Comment:
```suggestion
        if (result != null)
```

Reply:
Used this to stay consistent with the similar logic in events.js. @mcollina, before accepting this, is there a reason you went with the separate checks in events.js?